tests/pytorch_tests/model_tests/test_feature_models_runner.py

```diff
@@ -618,6 +617,13 @@ def test_gptq(self):
                               weights_quant_method=QuantizationMethod.UNIFORM,
                               params_learning=False).run_test()  # TODO: When params learning is True, the uniform quantizer gets a min value  > max value
 
+        # gradual activation quantization
```

Comment:
put it in a separate test function, this GPTQ test is already too long (takes very long time to run compared to other tests)